tests/sdk/nodejs/examples/ingress/index.ts

```diff
@@ -19,6 +19,93 @@ import { output as outputs } from "@pulumi/kubernetes/types";
 const ns = new k8s.core.v1.Namespace("test");
 const namespace = ns.metadata.name;
 
+// Install nginx ingress controller first
+const ingressNs = new k8s.core.v1.Namespace("ingress-nginx-ns", {metadata: {name: "ingress-nginx"}})
```

Comment:
Is the namespace name important? I normally try to avoid hardcoded names because they often cause problems in parallel test runs.

Reply:
No its not. Removed to be autonamed.